cue.mod/module.cue

```diff
@@ -0,0 +1 @@
+module: "github.com/grafana/grafana"
```

Comment:
This needs to be at the Grafana root so that internal imports - e.g., plugins importing core components, as happens in the table panel `models.cue` - doesn't cross a CUE module boundary.